Frends.SFTP.UploadFiles/Frends.SFTP.UploadFiles/Definitions/RenamingPolicy.cs

```diff
@@ -96,7 +98,7 @@ private string ExpandMacrosAndMasks(string originalFilePath, string filePath)
     private string ExpandFileMacros(string filePath)

```

Comment:
https://github.com/FrendsPlatform/Frends.SFTP/blob/8e43d1e628087198bc29a1405a75d9729decced8/Frends.SFTP.UploadFiles/Frends.SFTP.UploadFiles/Definitions/RenamingPolicy.cs#L61
This mehtod can be `static`.